README.md

```diff
@@ -23,21 +40,10 @@ docker build -t $USER/caliban_toolbox .
 
 ```
 
-The tensorflow version can be overridden with the build-arg `TF_VERSION`.
-
-```bash
-docker build --build-arg TF_VERSION=1.9.0-gpu -t $USER/caliban_toolbox .
-```
-
 ### Run the new docker image
 
 ```bash
-# NV_GPU refers to the specific GPU to run DeepCell Toolbox on, and is not required
-
-# Mounting the codebase, scripts and data to the container is also optional
-# but can be handy for local development
-
-NV_GPU='0' nvidia-docker run -it \
+nvidia-docker run -it \
```

Comment:
It may need a GPU if we intend on how we plan to make predictions to create annotations for users to correct in Caliban.